lapack.go

```diff
@@ -122,6 +122,12 @@ const (
 	EigBoth EigComp = 'I'
 )
 
+// JobEV specifies whether eigenvectors will be computed in Dsyev.
```

Comment:
I changed `SideEV` to `EVSide` and `JobEV` to `EVJob`. The rest I'll do in a next PR, I don't want this one to get too messy.